### ISSUE TITLE
Date range filter not working with function as data

### ISSUE THREAD

**Issue by khung** [vedmack/yadcf#262]
I'm trying to use the range_date filter with a function in either the columns.render property or the columns.data property of the DataTable. However, yadcf does not seem to work with either one. This seems related to issue #95 but the script does not even reach the changed code.

Using a function in the columns.render property gives incorrect filter results as yadcf uses the _aData value rather than the _aFilterData value:
```javascript
var table = $("#reqtable").DataTable( {
	"ajax": 'test.txt',
	"columnDefs": [
		{
			"targets": 0,
			"render": function ( data, type, row, meta ) {
				var jsDate = new Date(row[0] * 1000);
				return $.datepicker.formatDate("mm/dd/yy", jsDate);
			}
		}
	]
} );
yadcf.init(table, [
	{	column_number: 0,
		filter_type: "range_date"
	}
]);
```

Using a function in the columns.data property throws an error "dot_refs.split is not a function":
```javascript
var table = $("#reqtable").DataTable( {
	"ajax": 'test.txt',
	"columnDefs": [
		{
			"targets": 0,
			"data": function ( row, type, val, meta ) {
				if (type !== 'set') {
					var jsDate = new Date(row[0] * 1000);
					return $.datepicker.formatDate("mm/dd/yy", jsDate);
				}
			}
		}
	]
} );
yadcf.init(table, [
	{	column_number: 0,
		filter_type: "range_date"
	}
]);
```

**Comment by vedmack** [comment]
please provide a minimal jsbin that reproduces the issue to speed up the fix.

**Comment by khung** [comment]
I'm not certain if this is the proper way to create the jsbin, but this should be all the code needed:
https://jsbin.com/tihumilehu/edit?html,js
The columns.data example does not seem to work with DOM-sourced data, and JS Bin does not seem to have a way to get AJAX-sourced data. Hopefully you can test that in some other way.

**Comment by vedmack** [comment]
you can use a static file to feed the ajax source, just like I did in the [showcase](http://yadcf-showcase.appspot.com/DOM_Ajax_Multiple_1.10.html) see second table, you can get the sources of the [showcase from its github repo](https://github.com/vedmack/yadcf-showcase/blob/master/war/resources/js/dom_ajax_multiple_1.10_example.js#L37) .

Or you can also provide a link to your webapp

**Comment by khung** [comment]
Thanks for the hint. I've tweaked the jsbin to work with AJAX:
https://jsbin.com/samoxiyegi/edit?html,js,console,output
You can add/remove the leading / in the //* lines to comment or uncomment the columns.render and columns.data properties to see the behavior of either one.

**Comment by vedmack** [comment]
fixed in [0.8.9.beta.30](https://raw.githubusercontent.com/vedmack/yadcf/e324a9c06e58873c1cc9567cfec7ef669ef299c0/jquery.dataTables.yadcf.js) , see [jsbin sample](https://jsbin.com/jakuyuwita/1/edit?html,js,output)
